app/org/sagebionetworks/bridge/config/BridgeSpringConfig.java

```diff
@@ -211,6 +210,12 @@ public AmazonSimpleEmailServiceClient sesClient(BasicAWSCredentials awsCredentia
         return new AmazonSimpleEmailServiceClient(awsCredentials);
     }
 
+    @Bean(name = "sqsClient")
+    @Resource(name = "awsCredentials")
+    public AmazonSQSClient sqsClient(BasicAWSCredentials awsCredentials) {
+        return new AmazonSQSClient(awsCredentials);
```

Comment:
I'm not sure what credentials we're using here, but I didn't see SQS permissions on any of the app accounts...